src/lib/es5.d.ts

```diff
@@ -4277,20 +4277,34 @@ declare namespace Intl {
         supportedLocalesOf(locales: string | string[], options?: NumberFormatOptions): string[];
     };
 
+    type DateFormatNumericVerbosity = "numeric" | "2-digit";
+    type DateFormatNameVerbosity = "long" | "short" | "narrow";

```

Comment:
Please inline the type aliases. They're needlessly polluting the global namespace. And if the alias name makes it into quickinfo, it just obscures what the legal values are.

Reply:
Thanks, and addressed!